cafebabel/templates/base.html

```diff
@@ -90,34 +90,32 @@
   </div>
   <nav id=static-pages-links>
     {% set static_pages = static_pages_for(current_language) %}
-    <ul>
-      <li>
-        <ul>
-          <li><a href={{ static_pages.about }}>About</a></li>
-          <li><a href={{ static_pages.contact }}>Contact</a></li>
-        </ul>
-      </li>
-      <li>
-        <ul>
-          <li><a href={{ static_pages.vision }}>Editorial vision</a></li>
-          <li><a href={{ static_pages.faq }}>FAQ</a></li>
-        </ul>
-      </li>
-      <li>
-        <ul>
-          <li><a href={{ static_pages.terms }}>Terms and conditons</a></li>
-          <li>
-            <ul>
-              <li><a href=https://www.facebook.com/cafebabelenglish class=icon-facebook></a></li>
-              <li><a href=https://twitter.com/cafebabel_ENG class=icon-twitter></a></li>
-              <li><a href=https://www.instagram.com/inside.cafebabel class=icon-instagram></a></li>
-              <li><a href=https://www.youtube.com/channel/UCKanXxqYDt3vmz89Ig6RqBg class=icon-youtube></a></li>
-              <li><a href=https://www.linkedin.com/company/cafebabel-com class=icon-linkedin></a></li>
-            </ul>
-          </li>
-        </ul>
-      </li>
-    </ul>
+    <nav>
```

Comment:
Here I need a container to center the content (and `max-width: 75rem;`) but does it make sens to put a `<nav>` for you?